pages/common/exiftool.md

```diff
@@ -11,6 +11,10 @@
 
 `exiftool -All= {{file1 file2 ...}}`
 
+- Remove GPS EXIF metadata from given image files:
+
+`exiftool "-gps*=" {{image1 image2 ...}}`
```

Comment:
```suggestion
`exiftool "-gps*=" {{path/to/image1 path/to/image2 ...}}`
```

- Mandatory change: path format is [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).

```suggestion
`exiftool "-gps*=" {{path/to/image1.jpg path/to/image2.jpg ...}}`
```

- Mandatory change: if `exiftool` requires extensions to be passed then it [should](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#extensions) be shown here.
- Note: you can use any supported extension here but I recommend to stick to the most common ones.

Reply:
> As there are more then one fixes available for this line all of them are listed an you have to chose the most appropriate one.
> 
> * Mandatory change: path format is [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths).
> 
> * Mandatory change: if `exiftool` requires extensions to be passed then it [should](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#extensions) be shown here.
> * Note: you can use any supported extension here but I recommend to stick to the most common ones.

LGTM, After @EmilySeville7cfg's suggestions. I think we can go with the generalized one instead of using file extension for images as there seems to be support for multiple image formats https://exiftool.org/ .